media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz"
+CHECKSUM_SHA256="1976efd54a918eadd3cb10b34c77cee009e21ae56274148afa01edf32654e47d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	flac123 = $portVersion
+	cmd:flac123 = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libFLAC
+	lib:libao
+	lib:libogg
+	lib:libpopt
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libFLAC
+	devel:libao
+	devel:libogg
+	devel:libpopt
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	autoreconf -fi
```

Comment:
But that's flac's fault. The result will be that the test always passes.
Alternatively you can do this:

    TEST()
    {
        true # Sorry, this project has no tests!
    }

This allows us to remember that the project has no support. It can be useful, as in next years GCI, we may have tasks about researching which recipes would benefit from having a TEST block added, and we need to remember which one were already checked.